features/xhr.yml.dist

```diff
@@ -278,3 +278,59 @@ compat_features:
   - api.XMLHttpRequest.timeout
   - api.XMLHttpRequest.timeout_event
   - api.XMLHttpRequestUpload.timeout_event
+
+  # baseline: high
+  # baseline_low_date: 2016-08-02
+  # baseline_high_date: 2019-02-02
+  # support:
+  #   chrome: "16"
+  #   chrome_android: "18"
+  #   edge: "14"
+  #   firefox: "18"
+  #   firefox_android: "18"
+  #   safari: "6"
+  #   safari_ios: "6"
+  - api.ProgressEvent.ProgressEvent
+
+  # baseline: high
+  # baseline_low_date: 2016-08-02
+  # baseline_high_date: 2019-02-02
+  # support:
+  #   chrome: "37"
+  #   chrome_android: "37"
+  #   edge: "14"
+  #   firefox: "32"
+  #   firefox_android: "32"
+  #   safari: "8"
+  #   safari_ios: "8"
+  - api.XMLHttpRequest.responseURL
+
+  # baseline: high
+  # baseline_low_date: 2016-11-15
+  # baseline_high_date: 2019-05-15
+  # support:
+  #   chrome: "31"
+  #   chrome_android: "31"
+  #   edge: "12"
+  #   firefox: "6"
+  #   firefox_android: "50"
+  #   safari: "5.1"
+  #   safari_ios: "5"
+  - api.XMLHttpRequest.responseType
+
+  # baseline: high
+  # baseline_low_date: 2018-10-02
+  # baseline_high_date: 2021-04-02
```

Comment:
These new formdata keys became baseline hight quite a bit later than the other keys already in there. But they're still old enough that this shouldn't matter.